### DISCUSSION TITLE
Stucked in boot UEFI, when creating a windows 10 vm.

### DISCUSSION THREAD

**Discussion by vitali-droujko** [utmapp/UTM#4708] [UTM Help (macOS)] [answered]
I want a windows 10 vm on my macos using UTM.
I have download the right zip on (for amd64) 
https://uupdump.net/download.php?id=a8ee9740-2472-4a03-b85f-663dbbc21cce&pack=en-us&edition=core;professional
I have successfully run the script uup_download_macos.sh, that generate the iso.

<img width="1791" alt="Screenshot 2022-11-17 at 13 27 20" src="https://user-images.githubusercontent.com/71847509/202446243-1dc75488-3ec2-4f35-906c-6085ac82c562.png">
<img width="709" alt="Screenshot 2022-11-17 at 13 28 14" src="https://user-images.githubusercontent.com/71847509/202446525-616ba952-1efa-45f6-bfbe-1ce4ead63482.png">

**Configuration**  
* UTM Version: 4.0.9 (68)
* macOS Version: macOS ventura 13.1
* Mac Chip (Intel, M1, ...):  macbook pro 16" 2019 intel i7

**Crash log**  
App didn't crash they didn't started.

**Debug log**  
No debug log.

**Upload VM**  
?

**Comment by tuandatdavid** [comment]
It's broken ISO I think

**Comment by tuandatdavid** [comment]
I think you can download official windows 10 iso

**Reply by vitali-droujko** [reply]
I already tried to download the official windows 10 iso.
The file looks like : Win10_22H2_EnglishInternational_x64.iso

When I try to install it
I'm stuck here:
<img width="799" alt="Screenshot 2022-11-23 at 10 45 13" src="https://user-images.githubusercontent.com/71847509/203516251-9859478f-01b5-410c-a152-868e6d5db500.png">

**Comment by osy** [comment, marked as answer]
Try typing in `fs0:\efi\boot\bootx64.efi`

**Reply by vitali-droujko** [reply]
this is does not not work it's redirect me to this:
<img width="804" alt="Screenshot 2022-11-23 at 10 38 48" src="https://user-images.githubusercontent.com/71847509/203514715-427f80ee-cb35-4ddb-98d4-d132d5a1718d.png">

**Reply by osy** [reply]
And did you press any key???

**Reply by vitali-droujko** [reply]
Oh sorry, It's work !
